clang/lib/Interpreter/Interpreter.cpp

```diff
@@ -397,33 +397,48 @@ Interpreter::getOrcRuntimePath(const driver::ToolChain &TC) {
   std::optional<std::string> CompilerRTPath = TC.getCompilerRTPath();
   std::optional<std::string> ResourceDir = TC.getRuntimePath();
 
-  if (!CompilerRTPath) {
-    return llvm::make_error<llvm::StringError>("CompilerRT path not found",
-                                               std::error_code());
+  if (!CompilerRTPath && !ResourceDir) {
+    return llvm::make_error<llvm::StringError>(
+        "Neither CompilerRT path nor ResourceDir path found",
+        std::error_code());
```

Comment:
I'd keep this two checks separate and returning proper errors specific to the variable.